src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -132,7 +132,12 @@ function TableWrapper(props: TableWrapperProps) {
       </StyledTableContainer>
       {!constraints.active && (
         <FooterWrapper footerContainer={footerContainer} paginationNeeded={paginationNeeded}>
-          <PaginationContent {...props} handleChangePage={handleChangePage} isSelectionMode={isSelectionMode} />
+          <PaginationContent
+            {...props}
+            rect={rect}
```

Comment:
why not use the context selector inside PaginationContent?

Reply:
I like that. Will fix!